lomap/dbmol.py

```diff
@@ -85,6 +86,28 @@ def ecr(mol_i, mol_j):
     return scr_ecr
 
 
+def _find_common_core(mols, element_change: bool) -> str:
```

Comment:
Maybe a stupid question - how does this behave if one or more molecules in the dataset don't have a matching common core?

Reply:
the resulting "common core" is "", which is the default argument to findMCS for "no core given", so should be alright